### PR TITLE
Add provisioning-daemon package

### PR DESCRIPTION
**Pull request by mar3kk**
Signed-off-by: Marek Kubiczek <marek.kubiczek@imgtec.com>

### CODE REVIEW

File: provisioning-daemon/Makefile [hunk 1]
```diff
@@ -0,0 +1,55 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=provisioning-daemon
+PKG_VERSION:=1.0.0
+#PKG_VERSION:=dev
+PKG_RELEASE:=1
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=$(TOPDIR)/../../packages/provisioning-daemon
```
Comment (nikhil-zinjurde-imgtec) [inline provisioning-daemon/Makefile]: The source URL should be remote i.e. https://github.com/CreatorDev/ci40-provisioning-daemon.git

Developers should set the source link locally for development purpose

**Review by nikhil-zinjurde-imgtec — DISMISSED** [review]

File: provisioning-daemon/Makefile [hunk 2]
```diff
@@ -0,0 +1,57 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=provisioning-daemon
+PKG_VERSION:=v0.1.0
+PKG_RELEASE:=1
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:==https://github.com/CreatorDev/ci40-provision-daemon.git
```
Comment (ghost) [inline provisioning-daemon/Makefile]: Extra =

File: provisioning-daemon/Makefile [hunk 3]
```diff
@@ -0,0 +1,57 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=provisioning-daemon
+PKG_VERSION:=v0.1.0
+PKG_RELEASE:=1
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
```
Comment (ghost) [inline provisioning-daemon/Makefile]: No need to redefine PKG_BUILD_DIR. Remove it.

File: provisioning-daemon/Makefile [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  CATEGORY:=Utilities
+  TITLE:=Ci40 Provisioning Daemon
+  DEPENDS:=+letmecreate +ubus +ubox +libconfig
+endef
+
+define Package/provisioning-daemon/description
+  Imagination Technologies Ci40 provisioning daemon
+endef
+
+define Build/Compile
```
Comment (ghost) [inline provisioning-daemon/Makefile]: No need to define Build/Compile and Build/Clean if you call the default

File: provisioning-daemon/Makefile [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+define Package/provisioning-daemon/install
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/usr/lib/lua/creator
+	$(INSTALL_DIR) $(1)/etc/config
+	$(INSTALL_DIR) $(1)/usr/lib/lua/luci
+	$(INSTALL_DIR) $(1)/www/luci-static/resources
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/* $(1)/usr/bin
+	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/etc/config/provisioning_daemon $(1)/etc/config/provisioning_daemon
+	cp -r $(PKG_BUILD_DIR)/luci-mod-provisioning/luasrc/* $(1)/usr/lib/lua/luci
```
Comment (ghost) [inline provisioning-daemon/Makefile]: $(CP)

File: provisioning-daemon/Makefile [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+	cp -r $(PKG_BUILD_DIR)/luci-mod-provisioning/resources/* $(1)/www/luci-static/resources
+	cp -r $(PKG_BUILD_DIR)/usr/lib/lua/creator/* $(1)/usr/lib/lua/creator
```
Comment (francois-berder-imgtec) [inline provisioning-daemon/Makefile]: $(INSTALL_LIB) no ?

File: provisioning-daemon/Makefile (hunk 5 above)
Comment (mar3kk) [reply provisioning-daemon/Makefile]: done

File: provisioning-daemon/Makefile (hunk 4 above)
Comment (mar3kk) [reply provisioning-daemon/Makefile]: removed

File: provisioning-daemon/Makefile (hunk 3 above)
Comment (mar3kk) [reply provisioning-daemon/Makefile]: removed

File: provisioning-daemon/Makefile (hunk 2 above)
Comment (mar3kk) [reply provisioning-daemon/Makefile]: fixed

**Review by ghost — APPROVED** [review]

**Pull request merged**